src/processes/linux_list.rs

```diff
@@ -0,0 +1,44 @@
+use crate::processes::{ProcessInfo, ProcessList};
+use anyhow::Result;
+use std::collections::hash_map::Entry;
+use std::collections::HashMap;
+use std::path::PathBuf;
+use sysinfo::System;
+
+pub fn active_executables() -> Result<ProcessList> {
+    let mut executables: HashMap<PathBuf, ProcessInfo> = HashMap::new();
+    let mut sys = System::new_all();
+    sys.refresh_all();
```

Comment:
Why `_all` and not `_processes`?

Reply:
oh yes that makes sense (I didn't realize what `refresh_all` was doing in the first place)